### ISSUE TITLE
Math functions and Binary operators failing

### ISSUE THREAD

**Issue by JannisX11** [bridge-core/MoLang#5]
I tested a bunch of expressions, and found some, even simple ones, that cause the module to fail. I think besides solving the issues, it would be cool to have invalid expressions or expression parts return 0 instead of throwing an error, or to have that as a togglable option.

* Binary operator: `true ? 5`
* Any math functions: `Math.pow(2, 2)`

**Comment by solvedDev** [comment]
Thanks a lot for the super valuable feedback!

- AFAIK, `true ? 5` is actually an invalid statement because it has to be terminated with a semicolon in order to be parsed correctly. (Unless that changed by now; it was mentioned in an internal doc by the Minecraft team)
- I've added a variant of the execute function that catches errors. We didn't have this in before because bridge. uses errors for validating MoLang scripts. I realize that this is not ideal for other use cases.
- `Math.pow(2, 2)` runs fine for me. Can you elaborate how the math functions are broken for you? :)

**Comment by JannisX11** [comment]
I think a binary statement without semicolon should be valid. Not sure where they mentioned that it shouldn't, maybe you can point me to that. But I tested in-game and it worked fine, and the documentation mentions nothing about that.

Not sure why math doesn't work for me. I cloned the repository, installed dependencies, created a file with this content and ran it in Node:
```javascript
const MoLang = require('.');
let n = MoLang.execute('Math.pow(2,2)');
console.log(n);
```
And it threw this error:
```
TypeError: this.name.eval(...) is not a function
    at T.eval (C:\Users\Jannis\Documents\Development\MoLang\dist\main.cjs.js:1:8606)
    at Object.exports.execute (C:\Users\Jannis\Documents\Development\MoLang\dist\main.cjs.js:1:17339)
    at Object.<anonymous> (C:\Users\Jannis\Documents\Development\MoLang\test.js:2:16)
```

**Comment by solvedDev** [comment]
- It was part of the first draft of the new MoLang features inside of #partner_molang. Since you can get it to work without it, I'll remove the check for the semicolon. That's a super easy fix.
- Oh, that's an extremely good catch. Thanks, will be fixed in a sec 👍

**Comment by solvedDev** [comment]
Thanks again for the feedback. All of it should be addressed in v1.6.1! :)

Please note that I've changed the API surface to be a class in order to support better handling of execution environments and improve performance even further. Therefore, you'll need to change the JS accordingly:

```javascript
import { MoLang } from "molang"
const myMolang = new MoLang({ /** Variables, queries etc. here */ }, { /** Settings here */ })

let n = myMolang.execute('Math.pow(2,2)');
console.log(n);

// Or without errors
let n = myMolang.executeAndCatch('Math.pow(2,2)');
console.log(n);

```
